src/devices/cpu/uml.cpp

```diff
@@ -137,45 +137,49 @@ opcode_info const instruction::s_opcode_info_table[OP_MAX] =
 
 	// Control Flow Operations
 	OPINFO0(NOP,     "nop",      4,   false, NONE, NONE, NONE)
-	OPINFO1(DEBUG,   "debug",    4,   false, NONE, NONE, ALL,  PINFO(IN, OP, IANY))
+	OPINFO1(DEBUG,   "debug",    4,   false, NONE, NONE, ALL,  PINFO(IN, OP, IANY)) // MAME debugger breakpoint
+	OPINFO0(BREAK,   "break",    4,   false, NONE, NONE, NONE) // (for debugging) Hardware breakpoint for debugging the output assembly (will cause MAME to crash without a debugger attached)
```

Comment:
Comment on the second one is incorrect now – it should be downgraded to “may cause MAME to crash…” as it some platforms check for a debugger.